packages/design-system/src/helpers/defaultTheme.js

```diff
@@ -1,108 +1,97 @@
 const merge = require("lodash.merge");
+const helpers = require("./tokensHelpers/tokensHelpers");
 const colorTokens = require("./tokensHelpers/colorsTokensHelper");
-const radiusTokens = require("./tokensHelpers/radiusTokensHelper");
-const fontSizeTokens = require("./tokensHelpers/fontSizesTokensHelper");
-const spacingsTokens = require("./tokensHelpers/spacingsTokensHelper");
+const propertyTokens = require("./tokensHelpers/propertyTokensHelper");
+
 const colors = require("./tokens/tokensConfigs/colors.json");
+const fontSizes = require("./tokens/tokensConfigs/fontSizes.json");
 const borderRadius = require("./tokens/tokensConfigs/borderRadius.json");
 const spacings = require("./tokens/tokensConfigs/spacings.json");
 
 const colorPaletteConfig = colorTokens.getColorsConfig(colors);
-const borderRadiusConfig = radiusTokens.getRadiusConfig(borderRadius);
-const fontSizesConfig = fontSizeTokens.getFontSizesConfig(fontSizeTokens);
-const spacingsConfig = spacingsTokens.getSpacingsConfig(spacings);
+const borderRadiusConfig = propertyTokens.getPropertyConfig(borderRadius);
+const propertyConfig = propertyTokens.getPropertyConfig(propertyTokens);
```

Comment:
Shouldn't it be like this:
```suggestion
const fontSizesConfig = propertyTokens.getPropertyConfig(fontSizes);
```